contracts/Stake/StakeManager.sol

```diff
@@ -33,7 +32,6 @@ contract StakeManager is Initializable, Ownable {
     mapping(bytes32 => Token) tokenAddresses;
     MPondLogic MPOND;
     MPondLogic prevMPOND;
-    IClusterRegistry clusterRegistry;
```

Comment:
Storage change, should leave empty stub?

Reply:
right, will change this